tests/models/test_property_attachment.py

```diff
@@ -98,3 +104,27 @@ def test_add_with_properties(self):
             "new part",
             update_dict={self.property_name: self.project_root + "/requirements.txt"},
         )
+
+    # def test_save_as_attachment(self):
+    #     upload_path = os.path.join(
+    #         self.test_assets_dir,
+    #         "files",
+    #         "test_upload_image_to_attachment_property",
+    #         "test_upload_image.png"
+    #     )
+    #     self.property.upload(upload_path)
+    #
+    #     with temp_chdir() as target_dir:
+    #         filepath_xl = os.path.join(target_dir, 'image_xl.png')
+    #         self.property.save_as(filename=filepath_xl, size='XL')
+    #         path = os.path.join(target_dir, filepath_xl)
+    #         image_xl = Image.open(path)
+    #         self.assertEqual(image_xl.width, 2048)
+    #
+    #         filepath_sqs = os.path.join(target_dir, 'image_sqs.png')
+    #         self.property.save_as(filename=filepath_sqs, size='SQS')
```

Comment:
Would it be nice to shadow the enums from KE-chain here?
It also could be wise to reduce the image size that you are uploading for the sake of the tests.
The SQS is the same as a `size=h,w` so we could inject size=1,1 or something so we can upload a 4x4 image for testing :)

Reply:
```ATTACHMENT_IMAGE_SIZES = dict(
    SQXS=(100, 100),  # pixels square
    XS=(100,),  # pixels width
    S=(320,),
    SQS=(320, 320),
    M=(640,),
    SQM=(640, 640),
    L=(1024,),
    SQL=(1024, 1024),
    XL=(2048,),
    SQXL=(2048, 2048),
    XXL=(4096,),
    SQXXL=(4096, 4096)
)```

Reply:
or a 200 x 200 image and request a SQXS image (=(100, 100))

Reply:
Added a smaller image and adapted the test. Test still fails when recording cassettes, hence it is skipped until we can figure out why it's failing.